plugins/woocommerce/tests/e2e-pw/tests/merchant/order-status-filter.spec.js

```diff
@@ -55,26 +56,15 @@ test.describe(
 			await api.post( 'orders/batch', { delete: [ ...orderBatchId ] } );
 		} );
 
-		test( 'should filter by All', async ( { page } ) => {
-			await page.goto( '/wp-admin/admin.php?page=wc-orders' );
-
-			await page.locator( 'li.all > a' ).click();
-			// because tests are running in parallel, we can't know how many orders there
-			// are beyond the ones we created here.
-			for ( let i = 0; i < orderStatus.length; i++ ) {
-				const statusTag = 'text=' + orderStatus[ i ][ 0 ];
-				const countElements = await page.locator( statusTag ).count();
-				await expect( countElements ).toBeGreaterThan( 0 );
-			}
-		} );
-
 		for ( let i = 0; i < orderStatus.length; i++ ) {
 			test( `should filter by ${ orderStatus[ i ][ 0 ] }`, async ( {
 				page,
 			} ) => {
 				await page.goto( '/wp-admin/admin.php?page=wc-orders' );
 
 				await page.locator( `li.${ orderStatus[ i ][ 1 ] }` ).click();
+				// eslint-disable-next-line playwright/no-networkidle
+				await page.waitForLoadState( 'networkidle' );
```

Comment:
No selector we could wait for instead?  Like maybe something in the order listing?

Reply:
I shared some context in the comment [above](https://github.com/woocommerce/woocommerce/pull/51158#discussion_r1744621146). 

Regarding waiting for something, I tried with the status label and `class="current"` on top of the filter, but no luck. 
<img width="1799" alt="Screenshot 2024-09-05 at 01 44 04" src="https://github.com/user-attachments/assets/3595b28a-380b-4f9e-954f-25c80f5862d8">

Anyway, thanks for the feedback. Let me explore further, and try to find something else and avoid using `networkidle`.

Reply:
Okay, @lanej0, I think I got it with this commit [3012d88](https://github.com/woocommerce/woocommerce/pull/51158/commits/3012d884f056e57e182e9efd191039b7777c3eae). 

Thank you for pushing me that extra step! 🙌